### ISSUE TITLE
Add accessibility description to Notification controls

### ISSUE THREAD

**Issue by timusus** [timusus/Shuttle#26]
The Notification controls (referenced in MusicService.java), are missing their content descriptions, which is bad for accessibility.

These content descriptions need to be included (see PlayerFragment.java for reference)

**Comment by Farmerjoe12** [comment]
Can you elaborate on what notification controls you're talking about in this file?

**Comment by timusus** [comment]
`R.layout.notification_template_base` and its related files (including API level variations).

You'll see it referenced in `MusicService#1983`.
